models/movieDao.js

```diff
@@ -0,0 +1,45 @@
+const appDataSource = require('../models/dataSource');
+
+const getMovies = async (filter) => {
+  try {
+    const { main, release, sort } = filter;
+
+    let sortType = {
+      openingAsc: 'ORDER BY m.opening_date ASC',
+      openingDesc: 'ORDER BY m.opening_date DESC',
+    };
+
```

Comment:
해당 코드도 마찬가지로 수정되거나 변경이 되면 안되는 객체입니다. 따라서 let으로 선언하는게 아니라 const를 활용해야하고, 추가로 이럼에도 객체는 수정될 수 있기 때문에 상수 데이터 관리 목적과는 반대되는 경우가 발생할 수 있습니다. 따라서 JavaScript의 Object.freeze()를 활용합니다.
```suggestion
    const sortType = Object.freeze({
      openingAsc: 'ORDER BY m.opening_date ASC',
      openingDesc: 'ORDER BY m.opening_date DESC',
    });

```